hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -519,6 +542,23 @@ utxoFromTx (Tx body@(ShelleyTxBody _ ledgerBody _ _ _ _) _) =
 utxoMin :: Utxo -> Utxo
 utxoMin = Utxo . uncurry Map.singleton . Map.findMin . utxoMap
 
+findScriptOutput ::
```

Comment:
Isn't the returned value a (single) UTXO? I would have expected `Maybe TxOut` from the current name. i.e. naming this `findScriptUtxo` would be more intuitive to me.

Reply:
I merely kept the function as it was, just converted types to cardano-api :shrug:, I suppose some of them can be reviewed indeed.